common/utils/util.js

```diff
@@ -126,6 +128,16 @@ const getActionText = ({ description = '', intervention }, interventionList) =>
     ? interventionList.find(({ uuid }) => uuid === intervention).shortDescription
     : description
 
+const getCorrelationId = () => {
+  const { correlationId = '' } = cls.getNamespace(clsNamespace)
+  return correlationId
+}
```

Comment:
Could be a one liner. Still readable I think?
```
const getCorrelationId = () => getNamespace(clsNamespace).correlationId || ''
```

Both versions though don't cater for `getNamespace` returning undefined. I'm guessing thats not possible, but thought I'd mention :-)